components/admin_console/plugin_management/plugin_management.jsx

```diff
@@ -1045,7 +1032,44 @@ export default class PluginManagement extends AdminSettings {
                             onChange={this.handleChange}
                             setByEnv={this.isSetByEnv('PluginSettings.EnableMarketplace')}
                         />
-
+                        <BooleanSetting
+                            id='automaticPrepackagedPlugins'
+                            label={
+                                <FormattedMessage
+                                    id='admin.plugins.settings.automaticPrepackagedPlugins'
+                                    defaultMessage='Enable Automatic Prepackaged Plugins:'
+                                />
+                            }
+                            helpText={
+                                <FormattedMarkdownMessage
+                                    id='admin.plugins.settings.automaticPrepackagedPluginsDesc'
+                                    defaultMessage='When true, automatically installs any prepackaged plugin found to be enabled in the server configuration.'
```

Comment:
@aaronrothschild Might want to take a look at this text. I am not sure I would understand it if I was not already aware.

Reply:
@marianunez 
```suggestion
                                    defaultMessage='When true, the server will detect previously enabled plugins on the server and will automatically install them. '
```
Is that accurate?

Reply:
Yes, that is accurate. Will put a PR shortly.

Reply:
PR https://github.com/mattermost/mattermost-webapp/pull/4687